### DISCUSSION TITLE
smqtt with emqx

### DISCUSSION THREAD

**Discussion by SidcloudHit** [emqx/emqx#8550] [Q&A]
we are using IOT bases sensors ESP32 with arduino and we want to send data with SMQTT protocol with emqx. our server which will receive data using smqtt is rhel. we have installed emqx  with docker in our rhel server. Kindly share what process we need to follow or share the steps.

**Comment by zmstone** [comment]
Hi.
Do you mean the steps tow configure ssl listener in EMQX?
